### ISSUE TITLE
No error but not working

### ISSUE THREAD

**Issue by 5ervant** [jbyte/three-dragcontrols#2]
Does your build module working with the latest version of Three.js Node module?
Please check: https://stackoverflow.com/q/55558231/2007055

Many thanks!

**Comment by jbyte** [comment]
Hi,
the build module should work with the newest Three.js version (r103).
I have just updated the module (here on GitHub and in the NPM registry) so that the example uses the newest version.

Regarding your problem, I am not 100% sure if it's this but anyway in the line:
```
const dragControls = new DragControls(objects, sceneGl, rendererGl.domElement);
```
I assume the variable `sceneGl` is the type `THREE.Scene` but the class expects it to be `THREE.Camera` (or any derivative). But such a thing should return an error in the console so I'm not really sure.

Nothing else really looks off to me.

If that is not the problem, please create a jsfiddle that also has the bug, or provide a fuller code sample.

**Comment by 5ervant** [comment]
Yes, it was a mistake upon inputing `sceneGl` instead of `camera` but it doesn't fix the problem yet.
BUT it works when I commented out the code line to append the `rendererCss` on a `domElement`:

    // domElement.appendChild(rendererCss.domElement);

It seems `CSS3DRenderer` and `DragControls` are conflicting each other.

**Comment by jbyte** [comment]
I would have to take a look at the `CSS3DRenderer` and test a few examples to figure out what exactly is causing the conflict.

I'll get back to you when/if I figure it out.

**Comment by 5ervant** [comment]
I think `DragControls` have problem with `CSS3DRenderer`, see this post: https://discourse.threejs.org/t/dragcontrols-with-css3drenderer/783?u=5ervant
